core/policy/policy.go

```diff
@@ -172,3 +176,42 @@ func (p *policyChecker) CheckPolicyBySignedData(channelID, policyName string, sd
 
 	return nil
 }
+
+// CheckPolicyNoChannelBySignedData checks that the passed signed data are valid with the respect to
+// passed policy on the local MSP.
+func (p *policyChecker) CheckPolicyNoChannelBySignedData(policyName string, signedData []*protoutil.SignedData) error {
```

Comment:
Does this mean that all the elements in the `signedData` slice must comply with the policy?

Reply:
Yes, that's the intention.